Mopups/Mopups.Maui/Platforms/Android/Impl/AndroidMopups.cs

```diff
@@ -171,4 +172,28 @@ static Task<bool> PostAsync(Android.Views.View? nativeView)
 
         return tcs.Task;
     }
+    
+    static FrameLayout? GetTopFragmentDecorView()
+    {
+        if (Platform.CurrentActivity is not ComponentActivity componentActivity)
+        {
+            return null;
+        }
+
+        var fragments = componentActivity.GetFragmentManager()?.Fragments;
+        
+        if (fragments is null || !fragments.Any())
```

Comment:
since `fragments` is IList, we should use `Count` instead of LINQ for better perf.
```suggestion
        if (fragments is null || fragments.Count != 0)
```